lucida/weather/server/WeatherServer.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import sys
+sys.path.append('../')
+
+from lucidatypes.ttypes import QuerySpec
+from lucidaservice import LucidaService
+
+from thrift.transport import TSocket
+from thrift.transport import TTransport
+from thrift.protocol import TBinaryProtocol
+from thrift.server import TServer
+
+import urllib2
+import json
+import urllib
+
+port = 8088
```

Comment:
we really need to rethink how all the ports are set, it's super messy to have them nested deep into the code. then you need to worry about setting them correctly within the CMD center as well